internal/pkg/instrumentation/bpf/go.opentelemetry.io/otel/traceglobal/probe.go

```diff
@@ -41,6 +44,17 @@ func New(logger *slog.Logger) probe.Probe {
 		SpanKind:        trace.SpanKindClient,
 		InstrumentedPkg: pkg,
 	}
+
+	uprobeNewStart := &probe.Uprobe{
+		Sym:         "go.opentelemetry.io/otel/internal/global.(*tracer).newSpan",
+		EntryProbe:  "uprobe_newStart",
+		FailureMode: probe.FailureModeWarn,
```

Comment:
Should we have `PackageConstrainsts` here for >= 1.33?

Reply:
Updated. PTAL.